src/meta/embedded/src/meta_embedded.rs

```diff
@@ -92,7 +92,7 @@ impl MetaEmbedded {
     /// Initialize a global embedded meta store.
     /// The data in `path` won't be removed after program exit.
     pub async fn init_global_meta_store(path: String) -> Result<(), MetaStorageError> {
-        databend_common_meta_sled_store::init_sled_db(path);
+        databend_common_meta_sled_store::init_sled_db(path, 64 * 1024 * 1024 * 1024);
```

Comment:
64GB is too larger? cc @everpcpc for a check.

Reply:
The default value could be larger, and we can adjust it to small values in production.

Reply:
This is only used for embedded sled db, which is for testing purpose only.